### PR TITLE
chore: Add kubernetes doc

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

File: docs/teams/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+`envd` allows you to manage your development environments on Kubernetes. You can use `envd` command to create, delete, and update development environments.
+
+## Run envd-server on Kubernetes
+
+This feature requires a envd-server running on the Kubernetes. Thus you need to deploy it with the helm chart provided in the [envd-server](https://github.com/tensorchord/envd-server) repository:
+
+```bash
+$ git clone https://github.com/tensorchord/envd-server
+$ cd ./envd-server
+$ helm install envd-server ./manifests
```
Comment (terrytangyuan) [inline docs/teams/kubernetes.md:18]: We should publish the charts so users don't have to clone locally. It's pretty easy to set up with Github Actions.

Comment (terrytangyuan) [reply docs/teams/kubernetes.md:18]: Example: https://github.com/akuity/helm-charts

Comment (gaocegege) [reply docs/teams/kubernetes.md:18]: SGTM

Comment (gaocegege) [reply docs/teams/kubernetes.md:18]: https://github.com/tensorchord/envd-server/pull/73

Comment (terrytangyuan) [reply docs/teams/kubernetes.md:20]: Great!

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**